lib/recurly/client.php

```diff
@@ -141,6 +149,14 @@ private function _sendRequest($method, $uri, $data = '')
       curl_setopt($ch, CURLOPT_CUSTOMREQUEST, $method);
     }
 
+    if(Recurly_Client::$logger){
+      Recurly_Client::$logger->info('Send request to Recurly', array('method' => $method, 'uri' => $uri));
+      Recurly_Client::$logger->debug('Send request with headers', array('headers' => $headers));
+      if(!empty($data)){
+        Recurly_Client::$logger->debug('Recurly request body', array('body' => $data));
+      }
+    }
```

Comment:
I'm playing around with something like:
```
    if (Recurly_Client::$logger) {
      Recurly_Client::$logger->info('Recurly Request', array('method' => $method, 'uri' => $uri));
      Recurly_Client::$logger->debug('Request data', array('headers' => $headers, 'body' => $data));
    }
```
What do you think? I think seeing an `null` for missing headers or body would actually be helpful.